packages/list-store/src/search-params.test.ts

```diff
@@ -36,6 +37,18 @@ const sortMapping = {
   },
 };
 
+interface FilterKey {
+  name: string;
+  searchParamType: SearchParamTypes;
+}
```

Comment:
I'm not used to creating interfaces in tests (I normally test interfaces in tests). Is there a way to create the interface in the actual code and use it in the test?